packages/plugin-clientportal-ui/src/components/forms/General.tsx

```diff
@@ -291,87 +291,98 @@ function General({
 
   const renderFeatures = () => {
     return (
-      <Block>
-        <h4>{__('Features')}</h4>
-        {renderFeatureBlock(
-          'knowledgeBase',
-          <>
-            {renderControl({
-              label: 'Knowledge Base Name',
-              subtitle: 'Shown name on menu',
-              formValueName: 'knowledgeBaseLabel',
-              formValue: knowledgeBaseLabel,
-              placeholder: 'Please enter a label for Knowledge base'
-            })}
-            <FormGroup>
-              <ControlLabel required={true}>Knowledge base topic</ControlLabel>
-              <p>{__('Knowledge base topic in Client Portal')}</p>
-              <Select
-                placeholder="Select a knowledge base topic"
-                value={knowledgeBaseTopicId}
-                options={generateOptions(topics, '_id', 'title')}
-                onChange={handleSelectChange}
-              />
-            </FormGroup>
-          </>,
-          'kbToggle',
-          kbToggle || false
-        )}
-        {renderFeatureBlock(
-          'publicTask',
-          renderTaskPipelines(),
-          'publicTaskToggle',
-          publicTaskToggle || false
-        )}
+      (isEnabled('knowledgebase') ||
```

Comment:
@enkhtuvshinD check before render then if all 3 of plugins not enabled just return null